packages/components-native/src/AtlantisContext/AtlantisContext.tsx

```diff
@@ -52,6 +57,7 @@ export const defaultValues: AtlantisContextProps = {
   },
   floatSeparators: { group: ",", decimal: "." },
   currencySymbol: DEFAULT_CURRENCY_SYMBOL,
+  headerHeight: 50,
```

Comment:
Where does 50 come from? 
Is this something to do with `@react-navigation/stack`?

Reply:
No, it's just a random number, because the `headerHeight` differs from device to device and platform.

Reply:
Whoops, yes, it's the value of `useHeaderHeight` from `@react-navigation/stack`, but 50 is just a default number here

Reply:
I changed it to 0 because if the user don't pass this value to the context, we understand that actually there's no header to worry about. Does it make sense?

Reply:
Yes that makes more sense to me!